salt/states/firewalld.py

```diff
@@ -580,6 +583,11 @@ def _present(name,
                 ret['comment'] = 'Error: {0}'.format(err)
                 return ret
 
+    # if prune_services == None, set to True and log a deprecation warning
+    if prune_services is None:
+        prune_services = True
+        log.warning('prune_services currently defaults to True, but the default will be changed to False in a future release.')
```

Comment:
You will need to use the `salt.utils.warn_until` function

https://docs.saltstack.com/en/latest/topics/development/deprecations.html

Reply:
My bad, I'll use the `salt.utils.warn_until` function instead. would 'Fluorine' be the version to specify in the first argument?